### DISCUSSION TITLE
Detected zombie-worker error

### DISCUSSION THREAD

**Discussion by hjonathan2019** [coinfoundry/miningcore#1480] [General]
@oliverw hello, i have this error in miningcore, what it means?
![image](https://user-images.githubusercontent.com/59586038/197230622-c0889ca0-dab0-447f-b5d7-ba95cff16ffb.png)

**Comment by hjonathan2019** [comment]
why no one answer me?

**Reply by MiningCryptoLive** [reply]
It's not an error. See documentation. Session management for purging DDoS/flood initiated zombie workers

**Reply by hjonathan2019** [reply]
where is that documentation ????? i cant find it

**Reply by MiningCryptoLive** [reply]
> where is that documentation ????? i cant find it 

Did you read the readme.

![Screenshot_20221021-190111.png](https://user-images.githubusercontent.com/1049926/197303093-cd2370c4-a5ac-44b2-b8cc-9b80e8fadc80.png)

**Reply by hjonathan2019** [reply]
Yes, I saw that, but it doesn't explain anything, it doesn't explain how to do it, it doesn't explain where I can find a command, it doesn't explain how to use it...

**Reply by MiningCryptoLive** [reply]
There's no command. There's nothing to do. It's a built in feature!

**Reply by hjonathan2019** [reply]
ok, so i dont have to do anything with that red message?

**Reply by MiningCryptoLive** [reply]
No. It stops the zombie or ddos miner from accessing the pool

**Reply by hjonathan2019** [reply]
First of all, thank you very much for the answers.

second, it would have been much easier to reply "sir, you don't have to do anything, miningcore takes care of everything".

I understand that all of you developers are geniuses and know everything about it, but if people like me are here it's because we don't know everything.

I am currently testing with my version of miningcore, installed by me, if the project works I plan to switch to miningcore.pro (and buy the entire package)

**Reply by MiningCryptoLive** [reply]
That's fine. I am not a genius. I have learned just like you. Read and ask questions. I am not the owner of this software. I am a miner just like you.

**Reply by oliverw** [reply]
> First of all, thank you very much for the answers.
> 
> second, it would have been much easier to reply "sir, you don't have to do anything, miningcore takes care of everything".
> 
> I understand that all of you developers are geniuses and know everything about it, but if people like me are here it's because we don't know everything.
> 
> I am currently testing with my version of miningcore, installed by me, if the project works I plan to switch to miningcore.pro (and buy the entire package)

We'll try to improve our response times in the future. I apologize on behalf of the entire workforce that you had to wait for an answer or worse be bothered to educate yourself. We are sorry, it won't happen again.
